packages/core/test/Cardano/types/Cip1854ExtendedAccountPublicKey.test.ts

```diff
@@ -0,0 +1,61 @@
+import { Bip32PublicKeyHex } from '@cardano-sdk/crypto';
+import { Cip1854ExtendedAccountPublicKey } from '../../../src/Cardano';
+
+const bip32PublicKeyPrefix = 'acct_shared_xvk';
+
+describe('Cardano/types/Cip1854ExtendedAccountPublicKey', () => {
+  const publicKeyHex = Bip32PublicKeyHex(
+    '979693650bb44f26010e9f7b3b550b0602c748d1d00981747bac5c34cf5b945fe01a39317b9b701e58ee16b5ed16aa4444704b98cc997bdd6c5a9502a8b7d70d'
+  );
+
+  describe('Cip1854ExtendedAccountPublicKey', () => {
+    it('Accepts a valid bech32-encoded CIP1854 public key', () => {
+      const bip32PublicKey =
+        'acct_shared_xvk1q395kywke7mufrysg33nsm6ggjxswu4g8q8ag7ks9kdyaczchtemd5d2armrfstfa32lamhxfl3sskgcmxm4zdhtvut362796ez4ecqx6vnht';
+      expect(() => Cip1854ExtendedAccountPublicKey(bip32PublicKey)).not.toThrow();
+    });
+
+    it('Throws an error when an invalid bech32-encoded CIP1854 public key is passed', () => {
+      expect(() => Cip1854ExtendedAccountPublicKey(publicKeyHex)).toThrow();
+      expect(() =>
+        Cip1854ExtendedAccountPublicKey(
+          'addr_test1qpfhhfy2qgls50r9u4yh0l7z67xpg0a5rrhkmvzcuqrd0znuzcjqw982pcftgx53fu5527z2cj2tkx2h8ux2vxsg475q9gw0lz'
+        )
+      ).toThrow();
+      expect(() => Cip1854ExtendedAccountPublicKey('invalid')).toThrow();
+    });
+  });
+
+  describe('fromBip32PublicKeyHex', () => {
+    it('Encodes a valid CIP1854 bip32 public key according to CIP5 specification', () => {
+      const cip1854PublicKey = Cip1854ExtendedAccountPublicKey.fromBip32PublicKeyHex(publicKeyHex);
+      expect(cip1854PublicKey.startsWith(bip32PublicKeyPrefix)).toBe(true);
+    });
+
+    it('Throws an error when an invalid hex string is passed', async () => {
+      expect(() => Cip1854ExtendedAccountPublicKey.fromBip32PublicKeyHex(Bip32PublicKeyHex('invalid'))).toThrow();
+      expect(() =>
+        Cip1854ExtendedAccountPublicKey.fromBip32PublicKeyHex(
+          Bip32PublicKeyHex('701e58ee16b5ed16aa4444704b98cc997bdd6c5a9502a8b7d70d')
+        )
+      ).toThrow();
+    });
+  });
+
+  describe('toBip32PublicKeyHex', () => {
+    it('Decodes a bech32-encoded CIP1854 bip32 public key to a hex string', () => {
+      const cip1854PublicKey = Cip1854ExtendedAccountPublicKey.fromBip32PublicKeyHex(publicKeyHex);
+      const originalHex = Cip1854ExtendedAccountPublicKey.toBip32PublicKeyHex(cip1854PublicKey);
+      expect(originalHex).toEqual(publicKeyHex);
+    });
+
+    it('Throws an error when an invalid bech32 string is passed', () => {
+      expect(() =>
+        Cip1854ExtendedAccountPublicKey.toBip32PublicKeyHex(Cip1854ExtendedAccountPublicKey('invalid'))
```

Comment:
`Cip1854ExtendedAccountPublicKey` is what should be throwing here, not `Cip1854ExtendedAccountPublicKey.toBip32PublicKeyHex`. I suggest removing this test

Reply:
Fixed on 7b583d8

Reply:
Still testing `Cip1854ExtendedAccountPublicKey('addr_test1qvw383738')` [here](https://github.com/input-output-hk/cardano-js-sdk/pull/1346/files#diff-82112a97b58a6472f92ac1d7c91f6d618b21f04e545876d13c01e9e58ab918c7R49), but describe block says it should be testing `describe('toBip32PublicKeyHex'`

Reply:
Removed on db446c5